src/test/ui/stability-attribute/rustc-encodable-stability.rs

```diff
@@ -0,0 +1,8 @@
+// edition:2018
+#![allow(deprecated)]
+extern crate rustc_serialize;
```

Comment:
Nit: there should probably be a one-line gap here (and honestly I wish we had this as a lint), I sleepily misread this as the attribute being applied to the `extern crate` instead of the whole crate.